contrib/spotifyd.service

```
@@ -7,7 +7,10 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-ExecStart=/usr/bin/spotifyd --no-daemon
+User=spotifyd
+Group=spotifyd
+WorkingDirectory=/var/lib/spoty
```

forgot to fix that line as well?

```suggestion
WorkingDirectory=/var/lib/spotifyd
```

---

Indeed . Thank you